mavros_extras/src/plugins/mount_control.cpp

```diff
@@ -43,16 +44,34 @@ class MountControlPlugin : public plugin::PluginBase {
 		PluginBase::initialize(uas_);
 
 		command_sub = mount_nh.subscribe("command", 10, &MountControlPlugin::command_cb, this);
+		mount_orientation_pub = mount_nh.advertise<geometry_msgs::Quaternion>("orientation", 10);
 	}
 
 	Subscriptions get_subscriptions()
 	{
-		return { /* Rx disabled */ };
+		return {
+			make_handler(&MountControlPlugin::handle_mount_orientation)
+		};
 	}
 
 private:
 	ros::NodeHandle mount_nh;
 	ros::Subscriber command_sub;
+	ros::Publisher mount_orientation_pub;
+
+
+	/**
+	 * @brief Publish the mount orientation
+	 *
+	 * Message specification: https://mavlink.io/en/messages/common.html#MOUNT_ORIENTATION
```

Comment:
add the args description to the comment

Reply:
I think better to use `mo` instead of `o` name for message param.

Reply:
Is there anything left to do? I forgot to convert the angles to radians, which is done in the last commit.